index.html

```diff
@@ -23,15 +23,24 @@
         </div>
         <ul class="menu">
           <li class="nav-item"><a href="#portfolio">Portfolio</a></li>
-          <li class="nav-item"> <a href="#about">About</a></li>
-          <li class="nav-item"> <a href="#contact">Contact</a></li>
+          <li class="nav-item"><a href="#about">About</a></li>
+          <li class="nav-item"><a href="#contact">Contact</a></li>
```

Comment:
* Kindly fix this bug I noticed, when I click on the mobile menu it doesn't take me to the correct part of the page [Check requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#:~:text=When%20the%20user%20clicks%20(or%20taps)%20any%20of%20the%20mobile%20menu%20options%2C%20a%20correct%20part%20of%20the%20page%20is%20displayed.)

https://github.com/mulugithub/Microverse-Portfolio/assets/52497246/a302569f-281f-4ee0-a7a5-582203a08f44